### DISCUSSION TITLE
Modal click events

### DISCUSSION THREAD

**Discussion by therawk** [heroui-inc/heroui#1754] [Help]
Hey!
Really awesome Library.

I'm struggling with listening to the click events of the backdrop and the close button. Is it possible to listen to the events on those?

**Comment by jmchaves** [comment]
Create your custom button:

```
import {Modal} from "@nextui-org/react";

<Modal isOpen={isOpen} closeButton={<div><XMarkIcon className="h-7 w-7" onClick={() => close()}></XMarkIcon></div>}>
```
